client/components/domains/search-filters/dropdown-filters.jsx

```diff
@@ -65,32 +61,10 @@ export class DropdownFilters extends Component {
 	getFiltercounts() {
 		return (
 			( this.props.lastFilters.tlds?.length || 0 ) +
-			( this.props.lastFilters.exactSldMatchesOnly && 1 ) +
-			( this.props.lastFilters.maxCharacters !== '' && 1 )
+			( this.props.lastFilters.exactSldMatchesOnly && 1 )
 		);
 	}
 
-	getMaxCharactersValidationErrors() {
-		const {
-			filters: { maxCharacters },
-			translate,
-		} = this.props;
-		const isValid = /^-?\d*$/.test( maxCharacters );
-		return ! isValid ? [ translate( 'Value must be a whole number' ) ] : null;
-	}
```

Comment:
This method only checked if `maxCharacters` is an integer, so it can be removed.